schema/configuration.go

```diff
@@ -55,7 +55,15 @@ type DataSharing struct {
 	PushTo              []string
 }
 
-var MalformedConfigError = errors.New("config file is malformed")
+type MalformedConfigError struct {
```

Comment:
This does not need to be a public type. Make the first letter lowercase to make private. `malformedConfigError`